backend/app/JWT.py

```diff
@@ -0,0 +1,6 @@
+def get_jwt_key():
```

Comment:
This is slightly better than "TEST_VALUE_PLEASE_CHANGE" but we still cannot have the key in code at all - it'll have to loaded from a secure environment.

Reply:
For example, if we are going to be using AWS, then we'd want the QA and PROD _environments_ to have AWS Creds to pull the key from AWS SecretsManager. Dev, QA, and PROD would all have different keys.